app/templates/job-layout.html

```diff
@@ -40,6 +39,10 @@ <h2>Apply now</h2>
                 Wikimedia Deutschland commits itself to the principles of equal opportunity and does not discriminate on the basis of
                 ethnic origin, nationality, religion, political or other opinion, sex, age, disability or sexual identity or orientation.
             </p>
+
+            <p>
+                In light of an underrepresentation of women in our Software Development Department we strongly encourage qualified women to apply.                
```

Comment:
I am not a fan of this as I think the sex of a person should not matter in this context. No strong objections from me as long as one sex does not get preferential treatment in the hiring process.

Reply:
Well... we work in an industry that gives men all the opportunities to accumulate experience and doesn't really allow the same opportunities to women, so until there are equal opportunities between the genders, I say we definitely should encourage more women to apply and we should seek student / junior women developers to help and grow their skills in our department, so I actually like this line...

Reply:
I would perhaps go a bit further and encourage women to apply without the “qualified” modifier. Apparently, an internal study at HP found that women only applied for a promotion when they met 100% of the job requirements, while men would apply even if they only met 60% (I can’t find the primary source for this, but it’s mentioned in [this article](https://www.theatlantic.com/magazine/archive/2014/05/the-confidence-gap/359815/), and you can find more mentions with Google). Given that, I’m not a fan of the phrasing “encourage *qualified* women to apply”, when it seems likely that some men will apply when they might not be completely qualified. (For example, I joined here with exactly zero PHP experience.)

Reply:
I stand corrected, Lucas I totally agree with your comment.

Reply:
> I joined here with exactly zero PHP experience.
I personally think ability to learn is more important than things like experience in some random language and you are way overqualified in that as a person who worked with you on daily basis for months.

Reply:
About not requiring knowing a specific language: yes, that is important and held into account during the hiring process.

On equal opportunities: I think sex should not factor into that: so all sexes should have the same opportunity when they apply. Even if these opportunities are not equal in our industry then I think it would be immoral to discriminate in our hiring practices to somehow offset discrimination at other organizations. And if anything, since the tech industry as a whole is very left leaning, I strongly suspect that woman are given more opportunities than men. I've seen the same tendency at WMDE. If a woman does not quality it is often "can't we overlook this and that". I will not be part of such sex based discrimination.

Reply:
I agree with you @JeroenDeDauw that gender should not factor into that, but to fix that you'll need to invest a time machine, go back in time and pack the universities (that used to have men only / be biased for men) and fix them, also fix the perception that maths and engineering are "men" professions... after that fix that employment opportunities were given to men only (due to perspective of men that women should stay home) and other historical issues like that...

until we do that, we (the men in this profession) can only lend women the privilege that was given to us and try to level the playing field...

Reply:
"Discriminating against men" is one of very few terms like "discriminating against rich" which they say when they get taxed higher. When a group has less privileges and it gets special treatment because they had to go through lots of more trouble is not discriminating against the privileged group.